cypress/integration/unit/logic/schedule-parser/schedule.parser.spec.ts

```diff
@@ -54,14 +54,15 @@ fillWorkerInfo(shifts, employee_info, nurseSection, WorkerType.NURSE);
 fillWorkerInfo(shifts, employee_info, babysitterSection, WorkerType.OTHER);
 
 const expectedSchedule: ScheduleDataModel = {
-  schedule_info: { month_number: 10, year: 2020, daysFromPreviousMonthExists: true },
+  schedule_info: { month_number: 10, year: 2020 },
   shifts: shifts,
   month_info: {
     frozen_shifts: [],
-    children_number: dates.map((x) => 1),
+    children_number: dates.map(() => 1),
     dates: dates,
   },
   employee_info: employee_info,
+  isAutoGenerated: false,
```

Comment:
Musiałem dodać flagę do stanu, żeby sprawdzać czy grafik został automatycznie wygenerowany - czyli czy powstał tylko po to, żeby dopełnić inny miesiąć do pełnych tygodni. W takim grafiku wszyscy pracownicy mają ustawione zmiany na wolne